config/interface/output_element_series/industrial_heat_mekko.yml

```diff
@@ -120,13 +120,73 @@
   dependent_on: 
   output_element_key: industrial_heat_mekko
   key: industry_heat_backup_burner_network_gas_in_industrial_heat_network_mekko_supply_industrial_heat_mekko
-- label: industry_heat_burner_coal_in_industrial_heat_network_mekko
+- label: industry_heat_backup_burner_coal_in_industrial_heat_network_mekko
   color: "#333333"
+  order_by: 301
+  group: supply
+  show_at_first: 
+  is_target_line: false
+  target_line_position: 
+  gquery: industry_heat_backup_burner_coal_in_industrial_heat_network_mekko
+  is_1990: 
+  dependent_on: 
+  output_element_key: industrial_heat_mekko
+  key: industry_heat_backup_burner_coal_in_industrial_heat_network_mekko_supply_industrial_heat_mekko
```

Comment:
@kndehaan I think we can revert the changes in this file right?

Reply:
Yes, all changes are related to the backup burner.